pkg/infra/game/math/math.go

```diff
@@ -8,18 +8,21 @@ import (
 	"infra/config"
 )
 
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
```

Comment:
I might just not be seeing it on this PR but is init function being called somewhere?

Reply:
Check [here](https://www.developer.com/languages/inti-function-golang/)